runtime/jvmti/jvmtiThread.cpp

```diff
@@ -1382,27 +1397,32 @@ jvmtiSuspendResumeCallBack(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *o
 			J9JavaVM *vm = vmThread->javaVM;
 			J9VMThread *targetThread = NULL;
 			j9object_t carrierThread = (j9object_t)J9VMJAVALANGVIRTUALTHREAD_CARRIERTHREAD(vmThread, vthread);
+			U_64 isSuspendedFlag = J9OBJECT_U64_LOAD(vmThread, vthread, vm->isSuspendedInternalOffset);
+			BOOLEAN inTransition = J9_ARE_ANY_BITS_SET(isSuspendedFlag, ~0x1);
```

Comment:
using 0x1 makes it difficult to understand the logic. we should introduce macros and helper methods similar to what we do for `Continuation.state`.
```
#define J9_VIRTUALTHREAD_INTERNAL_STATE_SUSPENDED 0x1
#define J9_VIRTUALTHREAD_INTERNAL_STATE_CARRIERID_MASK (~(uintptr_t)0xff)
```